crates/holochain_conductor_api/Cargo.toml

```diff
@@ -59,6 +61,11 @@ sqlite-encrypted = [
   "holochain_zome_types/sqlite-encrypted",
   "holochain_keystore/sqlite-encrypted",
 ]
+schema = [
+  "dep:kitsune2_gossip",
```

Comment:
This is good and could be merged like this. The other way to do this would be to put `kitsune2_gossip/schema` in here which would both enable the optional dependency and turn the required feature on. It makes no difference right now, but it means that if some other feature wants to turn on the same dependency with other features then they can do so independently. 

Really not a big thing, and it can be changed later without this being a breaking change so I'm just going to approve. Just a minor thought :)

Reply:
No absolutely, you're right. It was bothering me a bit that the schema feature was "hard-coded" where the optional dependency is specified but I didn't know that I can have the best of both worlds. Changed it now.